src/main/java/com/github/manolo8/darkbot/backpage/entities/ItemInfo.java

```diff
@@ -10,9 +10,17 @@
     @SerializedName("name")
     private String name;
 
-    public ItemInfo(int lootId, String name) {
+    @SerializedName("T")
```

Comment:
all changes in here belong in a different PR. This PR is only for gg, this is from hangar, right?